test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+odoo14-addon-component @ git+https://github.com/OCA/connector@refs/pull/382/head#subdirectory=setup/component
+odoo14-addon-component_event @ git+https://github.com/OCA/connector@refs/pull/384/head#subdirectory=setup/component_event
+odoo14-addon-queue_job @ git+https://github.com/OCA/queue/pull/263/head#subdirectory=setup/queue_job
```

Comment:
addon name seems wrong: `s/_/-`

Reply:
@simahawk how do I know the name if it's not from the module folder name https://github.com/OCA/queue/tree/13.0/queue_job? Why with dash instead of underscore?

Reply:
Just tried it, same error

![image](https://user-images.githubusercontent.com/10927087/96639603-40170b80-1322-11eb-9a93-9469851557ca.png)

Reply:
the URL is wrong as well, compare it w/ the others right above ;)

BTW when you depend on other PRs please reference them in the description using a list of checkboxes to flag the ones that got merged. Example:

Depends on:

- [ ] https://github.com/OCA/queue/pull/263
...

Reply:
> @simahawk how do I know the name if it's not from the module folder name https://github.com/OCA/queue/tree/13.0/queue_job? Why with dash instead of underscore?

the name is exactly the same but with `-` instead of `_`. The format is `odoo{odoo_version}-addon-{normalized-addon-name}`

Reply:
> the URL is wrong as well, compare it w/ the others right above ;)

Silly mistake! Thanks to make me aware of it ;-)

Reply:
> BTW when you depend on other PRs please reference them in the description using a list of checkboxes to flag the ones that got merged. Example:
> 
> Depends on:
> 
> * [ ]  [OCA/queue#263](https://github.com/OCA/queue/pull/263)
>   ...

Done!

Reply:
> > @simahawk how do I know the name if it's not from the module folder name https://github.com/OCA/queue/tree/13.0/queue_job? Why with dash instead of underscore?
> 
> the name is exactly the same but with `-` instead of `_`. The format is `odoo{odoo_version}-addon-{normalized-addon-name}`.
> 
> Examples https://wheelhouse.odoo-community.org/oca-simple/ or https://pypi.org/search/?q=odoo13-addon

The name of the module was already correct it must be with underscore odoo14-addon-queue_job @ git+https://github.com/OCA/queue/pull/263/head#subdirectory=setup/queue_job
My mistake was just forgetting the "@refs" on the url. 
It's green now :-)
Thank you very much